README.md

```diff
@@ -45,6 +45,11 @@ When you become familiar to the TizenRT build system, you may use the *dbuild.sh
 As the build script is running based on Docker, it requires *sudo* for root permission.  
 To run Docker without *sudo*, refer to https://docs.docker.com/install/linux/linux-postinstall/.
 
+For windows environment, configuring CRLF is required as below.
```

Comment:
I think this is not a build step. Let's move this inside `Environment setup`, next of `2. Getting TizenRT source code`.